credentials/urls.py

```diff
@@ -83,3 +85,6 @@
     import debug_toolbar
 
     urlpatterns.append(re_path(r"^__debug__/", include(debug_toolbar.urls)))
+
+# Plugin django app urls
+urlpatterns.extend(get_plugin_url_patterns(PROJECT_TYPE))
```

Comment:
LGTM

Just a thought: I was wondering we'll get urls shadowing sometime... so plugins are able to "hijack" patterns...
But it must be en explicit tool (e.g. `get_plugin_url_pattern_overrides`).